archetype/pom.xml

```diff
@@ -152,4 +153,90 @@
             </extension>
         </extensions>
     </build>
+
+    <profiles>
+      <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-enforcer-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>enforce-maven</id>
+                                <goals>
+                                    <goal>enforce</goal>
+                                </goals>
+                                <configuration>
+                                    <rules>
+                                        <requireMavenVersion>
+                                            <version>[3.2.5,)</version>
+                                            <message>Maven 3.0 through 3.0.3 inclusive does not pass
+                                                correct settings.xml to Maven Release Plugin.</message>
```

Comment:
Maven versions 3.0 through 3.0.3 inclusive do not pass...

Reply:
I bumped the Java version and build on Jenkins, so now it passes